components/gitpod-db/src/typeorm/installation-admin-db-impl.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { inject, injectable, } from 'inversify';
+import { InstallationAdmin } from '@gitpod/gitpod-protocol';
+import { v4 as uuidv4 } from 'uuid';
+import { Repository } from 'typeorm';
+import { TypeORM } from './typeorm';
+import { InstallationAdminDB } from '../installation-admin-db';
+import { DBInstallationAdmin } from './entity/db-installation-admin';
+
+@injectable()
+export class TypeORMInstallationAdminImpl implements InstallationAdminDB {
+    @inject(TypeORM) typeORM: TypeORM;
+
+    protected async getEntityManager() {
+        return (await this.typeORM.getConnection()).manager;
+    }
+
+    protected async createDefaultRecord(): Promise<InstallationAdmin> {
+        const record: InstallationAdmin = {
+            id: uuidv4(),
+            sendTelemetry: false,
```

Comment:
Shouldn't it be `true` because it's “opt-out”?

```suggestion
            sendTelemetry: true,
```

Reply:
It will be `true` when the dashboard happens. See description:

> The sendTelemetry boolean currently defaults to `false` and will do until the admin panel exists then it will change to default `true`.

Reply:
Ah, makes sense. :+1: